.env

```diff
@@ -64,6 +64,7 @@ HASH_REDIS_HOST=localhost
 HASH_REDIS_PORT=6379
 
 HASH_OTLP_ENDPOINT=http://localhost:4317
+HASH_GRAPH_SENTRY_DSN=
```

Comment:
We should not set an empty default in `.env`

Reply:
commented out in [8e62433](https://github.com/hashintel/hash/pull/3091/commits/8e624331872d7bb6331f7e7bfd70a877aa182e66)